osfoffline/gui/qt/generated/preferences.py

```diff
@@ -129,8 +123,8 @@ def retranslateUi(self, Settings):
         self.textEdit_2.setHtml(_translate("Settings", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
 "<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
 "p, li { white-space: pre-wrap; }\n"
-"</style></head><body style=\" font-family:\'.Helvetica Neue DeskInterface\'; font-size:13pt; font-weight:400; font-style:normal;\">\n"
-"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Ubuntu\'; font-size:11pt;\">OSF Sync v" + VERSION + "</span></p>\n"
+"</style></head><body style=\" font-family:\'.SF NS Text\'; font-size:13pt; font-weight:400; font-style:normal;\">\n"
+"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Ubuntu\'; font-size:11pt;\">This is OSF OFFLINE. Please go ahead and use it and make more software based off of it. Please and Thank You. </span></p>\n"
```

Comment:
Thanks for pointing out.

On Wed, Jan 27, 2016 at 12:54 PM, Andy Boughton <notifications@github.com>
wrote:

> In osfoffline/gui/qt/generated/preferences.py
> <https://github.com/CenterForOpenScience/osf-sync/pull/140#discussion_r51021745>
> :
>
> > @@ -129,8 +123,8 @@ def retranslateUi(self, Settings):
> >          self.textEdit_2.setHtml(_translate("Settings", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
> >  "<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
> >  "p, li { white-space: pre-wrap; }\n"
> > -"</style></head><body style=\" font-family:\'.Helvetica Neue DeskInterface\'; font-size:13pt; font-weight:400; font-style:normal;\">\n"
> > -"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Ubuntu\'; font-size:11pt;\">OSF Sync v" + VERSION + "</span></p>\n"
> > +"</style></head><body style=\" font-family:\'.SF NS Text\'; font-size:13pt; font-weight:400; font-style:normal;\">\n"
> > +"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Ubuntu\'; font-size:11pt;\">This is OSF OFFLINE. Please go ahead and use it and make more software based off of it. Please and Thank You. </span></p>\n"
>
> Note that this change removes the version number from the preferences
> panel.
>
> (The manual edit got overwritten because this file is auto-generated)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf-sync/pull/140/files#r51021745>
> .
>